CircleciScripts/replace_android10.py

```diff
@@ -0,0 +1,45 @@
+from utils import replacefiles
+import demjson
+import sys
+import re
+import os
+root = sys.argv[1]
+
+replaces = [
+    {
+        "match" : 'android-23', 
+        "replace" : 'android-10',
+        "files" : [
+            "aws-android-sdk-core/build.gradle"
+        ]       
+    } ,
+    {
+        "match" : 'android-23.jar', 
+        "replace" : 'android-10.jar',
+        "files" : [
+            "aws-android-sdk-core/build.gradle"
+        ]       
+    } ,
+]
+replacefiles(root, replaces)
+
+
+
+
+AWSKeyValueStoreFile = os.path.join(root,"aws-android-sdk-core/src/main/java/com/amazonaws/internal/keyvaluestore/AWSKeyValueStore.java")
+newcontent = ""
+with open(AWSKeyValueStoreFile, 'r') as myfile:
+    content = myfile.read() 
+    pattern = r'if\s*\(apiLevel >= ANDROID_API_LEVEL_23\)\s*\{[^\}]+\}\s*else if\s*\(apiLevel >= ANDROID_API_LEVEL_18\)\s*\{[^\}]+\}'
```

Comment:
@kvasukib had added some annotations something along the lines of `@api23start` and `@api23end` would that be a better way to replace?